notebooks/240-yolox-object-detection/240-yolox-object-detection.ipynb

```diff
@@ -0,0 +1,4612 @@
azZyjuIzPUUgFVoL4FkRR6NmXZ34hHQhMJJBTXkLFjJGcmRhBCazWNh1pD2FOHE8nFpr5QoahsNm0s/TDSCypnVopArHA6QGLeCrGMm2Q4fMvX1HXIhLX6OJPWBoKU1FXjpS8WDPlUM7JRvbStgifrV1ZdWGemceJ3o8S8pZFv+SKlicWZCGmRMpR9IhaYbTDaUPlavlclF2ccxLCpw3Y/xIXmeUm/5N6jnGcZLlfO3Jl6fteDoS1uLhVP/I0yGyhhC5FZjFzzDnTttV/Rl+yiBsXiEsh/kfyABc7EAM5RbLSsozNosVZWDhMWTQaaRZ45+amCNYEcxbti18fJO+9CNOsvnRfdVM48GaFC0D48F3XrfCEfK9yWAmOmzC6sFmUHArjPPP69Tvevr7j8fBYOrlEXTfs91c8+/A9vtdu6ak46Ja73PDj+4Gf/P4rfvHx3+X4+IY8eXT0tGriBzeav/zRc25rTeyP9G/fcEqalKVLVMpJrrjWNI2V5Xdd0baNsHiMJqsku5acMUphTbVGKUAWRbWV4nI8HgVzV5q2qqmtYxhHfAzEWXY0m27L4nR96M+SUvnEZTqEUGiyBq0boXKCQJ5lAq4qxzzD4TQwjKIwfzjcY4wTV++UmMMRYyy7m1va7Za7128590eO/ZnZR2JIdN2GEKWApqpiu90wzTP3n38h2T/WloW4EESMFkV/6XNQSlFZi7MCpaKe2KUojSlW8gqFMkbcu1VhOirx/gveo3Li3J+Z/MzQT6Q5oGtLCoVqDLRdtxahBV5Z2FbLtVuw+XVS0hd37RDkwJ+maRXrhehXNGFp+hZmZyax2W4gc4GG83LQXnQ3C2VbmE6qPKeZRWmutUxCQPEYk0PT2gvLMpZnMhX4uOta5sKQquuFWFFIDvmirVlzaZ6EqS2EGGM0280GhUyjIUfZh5TCh9E0Xcf5dGKYhPgjxVgcwGMO0uj4WfZpRvRNkcToA19+/hXXux3GKlxtsVZcmbXJGA3GNVA3pdGQ6dw6yzSNhOC/YSGVQyR5L2dYVZV95sQSUrd6Dzph8y3x6yotwm6wShOTR2nRgamcefHslro0BH/W61tfZK6vr9abzXvBQPu+X3cpsvjVK169joqUB7TQEwVfXiyrATJxMZ1Uir7vL3h3+bsLZVShC91ZYSiZ3HHC2IaqckxjWK37p2laOylXcPS6qUsXKlPK4XgiJwSyUItTrl4PnJwyScvD0LYtrq5o6pq6bksXdslwWOAJEQDK9KLQ9P3I8ThwPPUMwygq9eGM1YZN29I2DS/ff4FrGrFZUZYxW07dNZ+bLb+8T/zo51/xyaeveLy/I4URlQI2DlzrM7/1ouE3n++oGBgfvuTLYSIBzlRURmGqqrgC6PXGlT2DHJht24gliZYlKznJXqOqV1sgHzwpBsYhYa0vtkHFV00qmBgmUsR+yCEY4sxU6JuJxOQvOxgRpopzxDxP4k1mhGk0J7maMXrqqqIfZ/pxkOlLFfIE4OdQlu6atm2pnYF5IMWJN3f3DNNMW7WcTz0/++OfcjqecVVFBibvGe8PhBBp6iJ+lKQgtDKlQw9idFkgNI1kwoAtth8XT67KubJn1Pi0QBtSVIP3HA6PnM7n4kSdwWicq1GuIqmMqWuiD6KWL5Y8EoAlEO2iEl/0HSvJJEmE9jzNpJzKhGDKgj4/eYZMQQtkKpAwPrO6LuQcBSK0ujSFYS0wC/SdKWJoii2/LtEACPvOGVOKSyJTsk7cxYJFGFmyV63Kz6W0omvrtYNPBVKV75tv7BkyrKzFaZ6Joex3tUapTLtpMFZz//hI8EHg+8IaPB5PjMOAMhpbOZnGQiSnQEhpDTuk7KRk8ZFRJO6Oj3z25ef83u/+jghvE7DEX2e1fg85B2Y/EkMqyEosjFFJDTXaYBpHKuagIUQUkmElU0+k65qCsOmy7BcBrrOWqpKpsHEVzrVUTYUx4kwRQubh4fBrz/FvdZFJKfHmzZu1w1lwQGFDLB1ntd74S+dl7cVCZdmpLNOCaA8cWklE6tJ1LQ/QMrksFvbGWGIKWCW2F7OfVvhmnkemaSjc84xzBue68qBoUhSvrugTkSCJhI2hbRuZxowsLlvb4FMSTn1dk5Moyi9eZE9giyz8d6M1KhvmSfYHk/ec+7ccDyfmcWb2k0w1lWO33XB9teHFzR5rHVOh7U4R5tmSrj/gqDf8/F3PT/74Nb/6/GccHo9E30NKaD+wyQc+bOHl1nBrFTo88PjqtRRU67BVUyAJOVyUMXSbhraRQ10IDIJpbxsRvCUlLK26btBGMU2jFJVFbFrVjFMWgWGKhFnsO3yMkveDYhgHuq4j5cQwCGEiKYhBrNe1syu9dyF01HXN4XBcJ9rzOKz07Wmayt6qoes2aKvp6oZaW+7uXuO90ITrpsbUhpgUOUbu3r3m0y9ec/d4oG1q2lbMNVcWlRIW2ul44vrZc5wVsopVGucM1iqaWijylatpCx3blAN0CXUjJRE1GkWMYsioUesS3UexOfLztFoZGWPXyX6BcL2X+3sqE4k2TyN8L7TqynwznmGxDbrsNBZjy8WDSyAx+TtqdTNYCAkO1gZx0aItfz7nxBJbsbDgQGyMBLoUavfyvC/kgpyl2dRaFThdDE5l9y3Pqi0sRWlGZVJaDmRpRl1R1ANcJhmtJYrDWs04DMIGQ1FXDZUVQsocZs76zDA6Tj5gq4qcMuP5zBwCrq4wWpGiJ6fi/K5AsQhmk0SRZdYCq6SO8dknn/P9737E82c3l2kwBoYSqyE07yIoTdJEbbdbQpIdUioT78JUy1H2bHEhvyTWTCCUiChMXdPVLfM4MIwnDseZv/R7v4NWmnGaOJ0GxnHi4XDP+TzyxRdvfu05/q0uMgJjdati/XQ6FWphtdqnLEv55aF4qjjWWpcHnUL9RLQqYbHVVvK5jLglwwUDXv6bCiarlKVta26aK6H7nQcy+RtU5wXzr6pKnJcRHnsIkbppZWFtNBjFbr+lkEp
```

Comment:
It is not recommended to include the model size comparison here, since the model size itself is not large in its raw format. Could focus on performance comparison only, with benchmark test results.